src/xdocs/config_coding.xml

```diff
@@ -4655,6 +4699,16 @@ for (String arg: args1) {
   &lt;property name=&quot;ignoreOccurrenceContext&quot; value=&quot;&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>Example:</p>
+        <div class="wrapper">
+          <pre>
+@SuppressWarnings("unchecked")
+    void myTest1(){}
+
+@SuppressWarnings("unchecked") // violation
```

Comment:
Please combine such example to other examples.
There should be the same example of Java code.
Only config is changing, and training comments in code.
To let user see how behavior is changing if property value is changing

Reply:
OK working on it